data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_teamster.json

```diff
@@ -8,7 +8,8 @@
     "attitude": 0,
     "mission": 3,
     "chat": "TALK_FREE_MERCHANT_TEAMSTER",
-    "faction": "free_merchants"
+    "faction": "free_merchants",
+    "traits": [ { "trait": "IGNORE_SOUND" }, { "trait": "NO_BASH" }, { "trait": "RETURN_TO_START_POS" } ]
```

Comment:
You need to move the traits to the NPC Class definition. They aren’t registered here.

Reply:
I'm aware. Just been too burnt out from work to fix it for a few days.